admin/admin-model.php

```diff
@@ -289,7 +298,7 @@ protected function validate_lang( $args, $lang = null ) {
 
 		// Validate slug is unique
 		foreach ( $this->get_languages_list() as $language ) {
-			if ( $language->slug === $args['slug'] && ( null === $lang || $lang->term_id !== $language->term_id ) ) {
+			if ( $language->slug === $args['slug'] && ( null === $lang || $lang->get_tax_prop( 'language', 'term_id' ) !== $language->get_tax_prop( 'language', 'term_id' ) ) ) {
```

Comment:
No need to change this as we kept the `term_id` property. For me the old way is more readable.